virtualcluster/pkg/syncer/conversion/helper.go

```diff
@@ -197,6 +197,15 @@ func (c *objectConversion) CleanOpaqueKeys(vc *v1alpha1.VirtualCluster, keyMap m
 	}
 }
 
+func WithSuperClusterLabels(labels map[string]string) map[string]string {
```

Comment:
I made it `With..` oriented to allow simple chaining, but wondering, if nil check is needed there and should we just use `ApplyLabels(labels)` instead, rather than chaining map. Any comments?